src/appmixer/google/drive/AddFileOrFolderPermission/component.json

```diff
@@ -0,0 +1,169 @@
+{
+    "name": "appmixer.google.drive.AddFileOrFolderPermission",
+    "author": "Appmixer <info@appmixer.com>",
+    "description": "Add a permission to a file or folder in Google Drive.",
+    "private": false,
+    "version": "2.0.0",
+    "auth": {
+        "service": "appmixer:google",
```

Comment:
it should be `appmixer:google:drive` since you've been using the auth.js located in the google/drive folder